R/map_peptides_on_structure.R

```diff
@@ -0,0 +1,342 @@
+#' Maps peptides onto a PDB structure
+#'
+#' Peptides are mapped onto PDB structures based on their positions. This is accomplished by replacing the B-factor information in the structure file with values that allow
+#' highlighting peptides when the structure is coloured by B-factor. In addition to only highlighting peptides, a continuous variable such as fold changes associated with peptides
+#' can be mapped onto the structure as a colour gradient.
+#'
+#' @param peptide_data a data frame containing at least the input columns to this function. If a structure file is provided in the \code{structure_file} argument,
+#' this data frame should only contain peptide and structure information for the provided structure since otherwise the information cannot be mapped correctly to the file due
+#' to the missing PDB identifier.
```

Comment:
please specify the minimally required columns, this might otherwise not be clear